getting_started/setup_linux/install_locally.rst

```diff
@@ -16,7 +16,7 @@ Make sure you have `set up your development environment <prepare_build_environme
 Build and install the kernel
 ****************************
 
-1. Change directory to ``~/sof/linux`` that you created on the setup page.
+1. Change directory to ``~/work/sof/linux`` that you created on the setup page.
```

Comment:
@bblacher do you mind changing the other occurrences of the same issue? see the 'remove the kernel' section

Reply:
Seems like I missed those. Fixed now.